packages/workflow/src/ErrorReporterProxy.ts

```diff
@@ -0,0 +1,28 @@
+export interface IReportingOptions {
+	tags?: Record<string, string>;
+	extra?: Record<string, unknown>;
+}
+
+interface ErrorReporter {
+	error: (error: Error, options?: IReportingOptions) => void;
+	warn: (warning: Error | string, options?: IReportingOptions) => void;
+}
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const instance: ErrorReporter = {
+	error: (error) => isProduction && console.error('ERROR', error.message, error.stack),
+	warn: (warning) => isProduction && console.warn('WARN', warning),
```

Comment:
Generally speaking, I'm never 100% sure when we should be logging to console vs using the logger.

Reply:
switched to logger.